pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -595,15 +595,14 @@ def root_gpu(self) -> Optional[int]:
 
     @staticmethod
     def _is_plugin_training_type(plugin: Union[str, Strategy]) -> bool:
```

Comment:
this needs update?

Reply:
Accelerator connector cleanup would be addressed in a separate PR as part of #10549